locales/en-US/webextension.properties

```diff
@@ -27,6 +27,8 @@ loginErrorDetails = We couldn’t save your shot because there is a problem with
 unshootablePageErrorTitle = We can’t screenshot this page.
 unshootablePageErrorDetails = This isn’t a standard Web page, so you can’t take a screenshot of it.
 selfScreenshotErrorTitle = You can’t take a shot of a Firefox Screenshots page!
+# Fired when someone makes a zero-width or zero-height selection
+emptySelectionErrorTitle = Your selection is too small
```

Comment:
This is a very obscure error: the user has the resize the selection down to zero height or width to encounter it, so I think if it doesn't get localized the impact will be very small.